src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -126,6 +135,37 @@ internal void DependencyRegen(WorkspaceModel ws)
         {
             RestartBanner.Visibility = Visibility.Hidden;
             var packageDependencies = ws.NodeLibraryDependencies.Where(d => d is PackageDependencyInfo).ToList();
+            var localDefinitions = ws.NodeLocalDefinitions.Where(d => d is LocalDefinitionInfo).ToList();
+
+            foreach (LocalDefinitionInfo info in localDefinitions)
+            {
+                try
+                {
+                    if (info.ReferenceType == ReferenceType.DYFFILE)
```

Comment:
I understand this is defined previously.. Comparing to  `DSFile`, can we rename this to `DyfFile` or `DYFFile` to be consistent?

Reply:
Updated it.